cypress/integration/ete-okta/reauthenticate.4.cy.ts

```diff
@@ -39,7 +39,7 @@ describe('Reauthenticate flow, Okta enabled', () => {
 					const sid = sidCookie?.value;
 					expect(sid).to.exist;
 					if (sid) {
-						cy.getCurrentOktaSession(sid).then((session) => {
+						cy.getCurrentOktaSession({ sid }).then((session) => {
```

Comment:
What's the effect of the brackets?

Reply:
I changed the first parameter of the `getCurrentOktaSession` method to now be an object instead of a value, so effectively i'm just turning the parameter i'm passing into `getCurrentOktaSession` an object with the following shape: 
```js
{
  sid: sid
}
```

This makes it easy to add or change parameters in the future, e.g. adding the `idx` cookie would become 
`.getCurrentOktaSession({ sid, idx })` or just `.getCurrentOktaSession({ idx }) instead of having to define multiple parameters e.g. `getCurrentOktaSession(undefined, idx)`